src/main/java/com/maxmind/geoip2/DatabaseReader.java

```diff
@@ -409,6 +413,51 @@ private Optional<AnonymousIpResponse> getAnonymousIp(
                 )
         );
     }
+    /**
+ * Look up an IP address in a GeoIP2 IP Risk.
+ *
+ * @param ipAddress IPv4 or IPv6 address to lookup.
+ * @return a IPRiskResponse for the requested IP address.
+ * @throws GeoIp2Exception if there is an error looking up the IP
+ * @throws IOException     if there is an IO error
+ */
+@Override
+public IPRiskResponse ipRisk(InetAddress ipAddress) throws IOException,
+        GeoIp2Exception {
+    Optional<IPRiskResponse> r = getIPRisk(ipAddress);
+    if (r.isEmpty()) {
+        throw new AddressNotFoundException("The address "
+                + ipAddress.getHostAddress() + " is not in the database.");
+    }
+    return r.get();
+}
+
+@Override
+public Optional<IPRiskResponse> tryIPRisk(InetAddress ipAddress) throws IOException,
```

Comment:
Minor style nit: our Java repos use camelcase for acronyms, initialisms, and abbreviations. If I recall correctly, this is what _Effective Java_ recommends and matches the style guidelines of several large orgs, e.g., [Google](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case). As such, this should be `tryIpRisk` and `getIpRisk` below. The Java standard library is unfortunately inconsistent on this.